android/app/src/main/res/layout/fragment_osm_profile.xml

```diff
@@ -135,6 +135,7 @@
         android:textColor="?colorAccent"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/osm_history"
-        app:layout_constraintVertical_bias="1" />
+        app:layout_constraintVertical_bias="1"
+        android:layout_marginBottom="60dp" /> 
```

Comment:
Is it a usual way to deal with such issues?
Or is it possible to make the label anchor to the actual dimensions of the system bar (which could be absent)?

Reply:
No this doesn't look right. Doing it this way results in an unnecessary gap at the bottom when the system bar is not shown.